centaur.nv.json

```diff
@@ -0,0 +1,218 @@
+        "start": 173,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 40
+      },
+      "06": {
+        "label": "Total Plays (Played and Free Games)",
+        "start": 180,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "07": {
+        "label": "Total Replays (Free Games)",
+        "start": 187,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "08": {
+        "label": "Game Percentage",
+        "start": 194,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "09": {
+        "label": "Total times 'High Score To Date' is beat",
+        "start": 201,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "10": {
+        "label": "Coins Dropped thru Coin Chute #1",
+        "start": 208,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "11": {
+        "label": "Coins Dropped thru Coin Chute #2",
+        "start": 215,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "12": {
+        "label": "Coins Dropped thru Coin Chute #3",
+        "start": 222,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "13": {
+        "label": "Number of Specials awarded from Panel Specials Only",
+        "start": 229,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "14": {
+        "label": "Number of minutes of Game Play",
+        "start": 236,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      },
+      "15": {
+        "label": "Number of Service Credits",
+        "start": 243,
+        "length": 7,
+        "encoding": "bcd",
+        "nibble": "high",
+        "min": 0,
+        "max": 99999
+      }
+    }
+  },
+  "adjustments": {
+    "High score functions": {
+      "01": {
+        "_note": "Increments by 10000",
+        "label": "Extra ball / Free game score level 1",
+        "start": 146,
+        "encoding": "bcd",
+        "nibble": "high",
+        "length": 6,
+        "min": 0,
+        "max": 999000,
+        "default": 0,
+        "scale": 10
+      },
+      "02": {
+        "_note": "Increments by 10000",
+        "label": "Extra ball / Free game score level 2",
+        "start": 152,
+        "encoding": "bcd",
+        "nibble": "high",
+        "length": 6,
+        "min": 0,
+        "max": 999000,
```

Comment:
This is correct.  min/max refer to the stored value and not the scaled/offset value.  I created #59 to track that documentation update.